### ISSUE TITLE
3D Model Source Files

### ISSUE THREAD

**Issue by jrhodnik** [topolittle/nitrous-setup-rx8#1]
Greetings!

Could you possibly upload the model files as STEP? I'd like to modify your design to instead hold some normal 12v accessories, similar to [this one](https://www.thingiverse.com/thing:2720594). Unfortunately, that ones's for a newer RX8 so it's not compatible with mine. I plan to upload the modified version to Thingiverse after it's complete, along with model source files as well.

Thank you kindly!

**Comment by topolittle** [comment]
I don't have access to my Solidworks installation at the moment, maybe in a
few days.
However, if you are able to open Solidworks files, I can share the full
project with you.

Stephane

**Comment by jrhodnik** [comment]
I have access to both Fusion 360 and Onshape, both of which apparently have support for Solidworks files.

Thank you for your response!

**Comment by topolittle** [comment]
Hi John,

I tried to send you the project by email but the file is too large. So here
is the file on my Dropbox:
https://www.dropbox.com/s/ot5q8igk4lx34mj/ashtray_solidworks.zip?dl=0

The parts you are looking for are front_plate.sldprt and pod_holder.sldprt.
You can also open the file fron_panel.sldasm which is an assembly of the
two parts.

Let me know if you have any questions. In the meantime I will try to upload
the steps files to Thingiverse.

Stephane

**Comment by jrhodnik** [comment]
Hey Stephane,

This worked great! I was able to convert the `sldprt` files to `STEP` files using Onshape's free trial. It is also apparently possible using Fusion 360, but you gotta pay for that. And I'm just a hobbyist, so it didn't make much sense for me to do so.
[For anyone who needs the `STEP` files, here they are.](https://github.com/topolittle/nitrous-setup-rx8/files/9308835/Converted.zip)

I did modify them and made a simple flat template that can be modified with whatever people would like, but I'm unable to test it as I don't actually have an ash tray for my car just yet. The goobers that owned it before me ripped a bunch of stuff out of the interior 😞. Once I get an ashtray I'll give it a test.

Onwards and upwards!
